src/main/java/Contacts0.java

```diff
@@ -146,4 +144,12 @@ public static void main(String[] args) {
         }
     }
 
+    private static void showWelcomeScreen() {
+        System.out.println("|| ===================================================");
+        System.out.println("|| ===================================================");
+        System.out.println("|| Contacts - Version 0.0");
+        System.out.println("|| Welcome to Contacts!");
+        System.out.println("|| ===================================================");
+    }
```

Comment:
nice changes, pls complete this for the rest of the code and methods. by tonight, thanks

Reply:
Understood. Thanks